spec/models/search/collection_decorator_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "spec_helper"
+
+describe CollectionDecorator do
+  
+  let!(:collection) { create(:collection) }
+  let!(:search_results) do
+    [{
+      "_id" =>
+      collection.id.to_s,
+      "_source" => {
+        "id" => collection.id.to_s,
+        "name" => collection.name,
+        "title" => collection.title,
+        "description" => nil,
+        "created_at" => collection.created_at.to_s,
+        "parent_id" => nil,
+        "closed" => false,
+        "unrevealed" => false,
+        "anonymous" => false,
+        "owner_ids" => collection.all_owners.pluck(:id),
+        "moderator_ids" => [],
+        "signup_open" => nil,
+        "signups_open_at" => nil,
+        "signups_close_at" => nil,
+        "assignments_due_at" => nil,
+        "works_reveal_at" => nil,
+        "authors_reveal_at" => nil,
+        "general_fandom_ids" => [],
+        "public_fandom_ids" => [],
+        "general_fandoms_count" => 5,
+        "public_fandoms_count" => 10,
```

Comment:
Are we still indexing these? I think we might've stopped, at least with the counts.